Cannot build deb on debian8 without editing Makefile

##### ISSUE TYPE
 - Bug Report
##### COMPONENT NAME
make deb fails

##### ANSIBLE VERSION
Clone of ansible/ansible as of today. a0344acd78c422457b5a018b9de59e213f39dc87

##### CONFIGURATION
docker run -ti debian:8 /bin/bash
apt-get install the following
git, make, python-sphinx, python-yaml, asciidoc, python-setuptools
git clone 
https://github.com/ansible/ansible
cd ansible
make deb
##### OS / ENVIRONMENT
See above, debian 8 docker image
##### SUMMARY
When running make deb on debian 8, you get the following error.
```
Traceback (most recent call last):
  File "./docs/bin/generate_man.py", line 128, in <module>
    f.write(manpage)
UnicodeEncodeError: 'ascii' codec can't encode character u'\xa9' in position 4964: ordinal not in range(128)
Makefile:306: recipe for target 'docs' failed
make: *** [docs] Error 1
```
I was able to work around this, by removing the doc section in the Makefile, but this is not an ideal long term solution.
##### STEPS TO REPRODUCE
See configuration section.

##### EXPECTED RESULTS
Produces a deb

##### ACTUAL RESULTS
Throws a traceback

--- a/docs/bin/generate_man.py
+++ b/docs/bin/generate_man.py
@@ -1,10 +1,11 @@
 #!/usr/bin/env python
 
 import os
-import io
 import sys
 
 from jinja2 import Environment, FileSystemLoader
+
+from ansible.module_utils._text import to_bytes
 
 
 def get_options(optlist):
@@ -124,6 +125,6 @@ if __name__ == '__main__':
 
         manpage = template.render(tvars)
         filename = '../man/man1/%s' % output[libname]
-        with io.open(filename, 'w') as f:
-            f.write(manpage)
+        with open(filename, 'wb') as f:
+            f.write(to_bytes(manpage))
             print("Wrote man docs to %s" % filename)

